.gitignore

```diff
@@ -2,3 +2,7 @@
 *.log
 
 node_modules
+
+# resin app files for resin sync
+dist/server.js
+dist/start.sh
```

Comment:
I don't think we should add personal setup specific ignores to the repo (unless they're not?)

Reply:
Correct, this is a leftover from the development process